x/subspaces/types/subspace.go

```diff
@@ -66,6 +88,11 @@ func (sub Subspace) Validate() error {
 		return fmt.Errorf("invalid subspace type: %s", sub.Type)
 	}
 
+	validLogo := commons.IsURIValid(sub.Logo)
+	if !validLogo {
```

Comment:
This can be simplified and should also contemplate empty URIs to represent no image: 

```suggestion
	if strings.TrimSpace(sub.Logo) != "" && !commons.IsURIValid(sub.Logo) {
```